### ISSUE TITLE
Events map/Celebrations list

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/wp20.wordpress.net#3]
Designs

| Desktop | Mobile |
|-|-|
| ![What's on_](https://user-images.githubusercontent.com/1017872/225156558-4f8c2037-167b-46a1-b4e3-036080cbc465.png) | ![What's on_ — Mobile](https://user-images.githubusercontent.com/1017872/225156609-a308ab55-67c1-4783-836f-9161d33479be.png) |

Get working locally:

- [x] Update `wp15` plugin with new dates, etc, and rename to `wp20`
- [x] Update to latest `Meetup_Client` and use new API/data formats
- [x] List events on `What's On` page 
- [x] Make the events map on the `What's On` page functional
- [ ] Style map and events content

Deploy:

This shouldn't make it visible to users yet, b/c #6 is needed for that. It should get everything in place though, so that we can flip it on when the design is ready.

- [x] Wait for #8 to be done
- [x] Setup auth for events plugin on production
- [x] Setup page 
- [x] Rename theme template to match `What's On` title - will be done in #6 
- [x] Make `What's On` the homepage - Wait until the theme is finished and deployed?
- [x] Update content on `Celebrations` page to no longer list events, because they're automated on `What's On` now. Maybe add link to homepage so see events.

**Comment by iandunn** [comment]
Page stub: https://wp20.wordpress.net/wp20-celebrations/

**Comment by iandunn** [comment]
Actually, it looks like the Figma has a `What's On` page, similar to WP15's homepage. It seems like the list/map should appear there, and I guess we'll then delete the current "Celebrations" page? 

Does that sound right  @marko-srb, @catederosia, @eidolonnight?

**Comment by adamwoodnz** [comment]
> Actually, it looks like the Figma has a `What's On` page, similar to WP15's homepage. It seems like the list/map should appear there, and I guess we'll then delete the current "Celebrations" page?
> 
> Does that sound right @marko-srb, @catederosia, @eidolonnight?

I have a `page-whats-on` template on the way https://github.com/WordPress/wp20.wordpress.net/pull/6/files#diff-d749d882c0b9bd6bf88faa23777f8db6c51500a04a117570e54ba1d0f4905f14

**Comment by iandunn** [comment]
Perfect!

**Comment by iandunn** [comment]
I setup the page on production, and manually added the shortcode so we can see it in the meantime:

https://wp20.wordpress.net/whats-on/

@catederosia, let me know if there are any events missing from that list.

**Comment by pablohoneyhoney** [comment]
Some relevant notes [here](https://github.com/WordPress/wp20.wordpress.net/issues/4#issuecomment-1470481955) regarding wayfinding, information density, etc. Some additional notes:  

- In mobile, the map could be full bleed, the margins make it slightly narrow.
- Search should be above the map. 
- Ideally, the listing rearranges by proximity (which I think it isn't happening in wp15).
- We should get the listing to be much more compressed so the scroll is not as long.
- The hierarchy in each event isn't as clear as wp15, which has the title and the link clearly visible. Here all is almost at the same visual level. 

<img width="593" alt="Screen Shot 2023-03-15 at 7 12 46 PM" src="https://user-images.githubusercontent.com/25516611/225404189-b6c629f0-3719-4f77-8bc7-f1dbfeef616e.png">

- Verbally, the sequence isn't flowing well yet. They seem disconnected and decontextualized instead of flowing from one to the other. Also _Join us_ could be more explicit to what we want people to _join_.

<img width="767" alt="Screen Shot 2023-03-15 at 7 14 48 PM" src="https://user-images.githubusercontent.com/25516611/225404618-ce2315d1-4994-4504-be51-5b2be95452c1.png">


- This is general, but let's not center align the texts, by default use left alignment, especially in longer texts (see mobile). 
- Afaik, all events are in Meet Up, but feels abrupt to link to there without a bit of context. Again, nothing long but it's worth explaining what the listing is a bit clearer. _Join the meet ups that are happening this year all over the world._ Or alike.

**Comment by iandunn** [comment]
We haven't started doing any design work on this issue yet, and the production site currently just has a placeholder theme, so IMO it'd be best to wait until we're finished to review anything.

**Comment by iandunn** [comment]
Er, are those comments about the Figma, or the current implementation? If it's the Figma, it's going to be hard for us to interpret and implement them perfectly just from words, so it'd be best to make the changes in Figma and then we can build off of that.

**Comment by pablohoneyhoney** [comment]
I was commenting on the designs so far.

**Comment by StevenDufresne** [comment]
Has this been unblocked?

**Comment by adamwoodnz** [comment]
> Has this been unblocked?

I've asked in Figma https://www.figma.com/file/3veAfpzTQ82v8G3sbvaJZh?node-id=1476:18297#405176156

**Comment by adamwoodnz** [comment]
With only 7 weeks to go, we can continue and iterate if needed. Final designs:

| Desktop | Mobile |
|-|-|
| ![What's on_](https://user-images.githubusercontent.com/1017872/229393122-c799913c-2fc6-4108-9042-12bb9fa7919f.png) | ![What's on_ — Mobile (1)](https://user-images.githubusercontent.com/1017872/229393040-1d9db2a6-89ce-4e39-a869-9b80d557cca9.png) |

**Comment by iandunn** [comment]
#40 and #48 are merged now, which covers most of this. Those are pushed to the staging site for live review.

#50 is in progress for the only other issues I'm aware of. 

I'll go ahead and close this since it's gotten fairly long, and we can open a new issue if there's any iteration needed.
